styles/globals.css

```diff
@@ -17,7 +17,7 @@
 }
 
 body {
-	background-color: #282c34;
+	background-color: var(--purple-900);
```

Comment:
The colour variables are not there to be used directly, you have to create other colour variables that use the base variables and have a specific purpose (e.g. background-color)